archstrike-testing/libsocket-git/PKGBUILD

```diff
@@ -0,0 +1,36 @@
+# Maintainer: ArchStrike <team@archstrike.org>
+
+buildarch=220
+
+_pkgname=libsocket
+pkgname=${_pkgname}-git
+pkgver=r546.8332600
+pkgrel=1
+pkgdesc="The ultimate socket library for C and C++"
+arch=('i686' 'x86_64' 'armv6h' 'armv7h' 'aarch64')
+url="https://github.com/dermesser/${_pkgname}"
+license=('custom')
+groups=('archstrike')
+depends=('gcc-libs')
+makedepends=('cmake' 'git')
```

Comment:
while replacing regular packages with -git packages, we add:
```
replaces=('name_of_old_pkg')
provides=('name_of_old_pkg')
conflicts=('name_of_old_pkg')
```
to avoid issues during system updates.